behaviors/smart-contracts/system/_Committee.md

```diff
@@ -0,0 +1,67 @@
+# `_Committee` System Service
+
+The system service charged with maintaining validators reputation (based on accepted block proposals) and for obtaining an ordered list of validators - committee.\
+Implemented on the `Native` processor.\
+Note: Committee members coincide with elected validators (obtained by calling `_Elections.getElectedValidatorsOrbsAddress()`).\
+
+#### Permissions
+* This is a system contract that runs under `System` permissions.
+
+
+#### State
+* Constants:
+    * ToleranceLevel (4)
+    * ReputationBottomCap (10)
```

Comment:
please add the motivation for selecting these numbers, both so we understand the selection and be able to understand how to approach changing them